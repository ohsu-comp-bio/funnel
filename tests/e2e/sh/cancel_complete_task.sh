id=`funnel run 'echo hello'`
funnel task wait $id
funnel task cancel $id
funnel task get --view MINIMAL $id
