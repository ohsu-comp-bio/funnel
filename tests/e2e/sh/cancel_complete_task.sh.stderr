Error: [STATUS CODE - 500]	{"error":"Won't switch between two terminal states: COMPLETE -\u003e CANCELED","code":2}
